### ISSUE TITLE
Accessibility Issues in Footer

### ISSUE THREAD

**Issue by peteredwards** [uol-library/spacefinder-ui#25]
**WCAG Criterion 2.4.7 Focus Visible (Level AA)**

* The footer occasionally partially, and perhaps fully, obscures the focus indicator as you tab through the filter options/checkboxes.

**WCAG Criterion 2.5.3 Label in Name (Level A)**

* The visually presented ‘2022 University of Leeds’ link in the footer has the label of ‘Copyright information - About this site’ – the label doesn’t include the visual text. Perhaps the two should be separated at the label applied to a new visual link?

**Comment by peteredwards** [comment]
Fixed visibility issue with filter options in 3394e16b67f31c39c69e0e4107fe146860a3e592

Fixed footer link text in e87d18f03396e572313b2a3bc8c9ecdcda2b8274
